### PR TITLE
Added support for SQS event types

### PR DESCRIPTION
**Pull request by jackholmes1010**
Added support for SQS event types, defined using `@SQS("arn")`.

Updated the example Serverless project.

### CODE REVIEW

**Review by johnmikk — APPROVED** [review]
:fire::fire:

File: tests/testUtil.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-import { APIGatewayProxyEvent, Context } from "aws-lambda"
+/* eslint-disable @typescript-eslint/ban-types */
+import { APIGatewayProxyEvent, Context, SQSEvent } from "aws-lambda"
 
-export const createAPIGatewayEvent = ({
+export const createApiGatewayEvent = ({
```
Comment (johnyherangi) [inline tests/testUtil.ts]: `createAPIGatewayEvent` seems more consistent with the rest of the project (uppercase abbreviations)

File: tests/testUtil.ts [hunk 2]
```diff
@@ -66,7 +67,27 @@ export const createAPIGatewayEvent = ({
     return eventTemplate
 }
 
-export const createAPIGatewayContext = (): Context => ({
+export const createSqsEvent = (...arns: string[]): SQSEvent => ({
```
Comment (johnyherangi) [inline tests/testUtil.ts]: `createSQSEvent` seems more consistent with the rest of the project (uppercase abbreviations)

File: tests/testUtil.ts [hunk 3]
```diff
@@ -66,7 +67,27 @@ export const createAPIGatewayEvent = ({
...
+            ApproximateFirstReceiveTimestamp: "",
+        },
+        messageAttributes: {},
+        md5OfBody: "",
+        eventSource: "aws:sqs",
+        awsRegion: "",
+    })),
+})
+
+export const createApiGatewayContext = (): Context => ({
```
Comment (johnyherangi) [inline tests/testUtil.ts]: `createAPIGatewayContext` seems more consistent with the rest of the project (uppercase abbreviations)

**Review by johnyherangi — APPROVED** [review]

**Pull request merged**